cloudbuild.yaml

```diff
@@ -6,8 +6,9 @@ steps:
 # see https://github.com/kubernetes/test-infra/tree/master/config/jobs/image-pushing
   - name: gcr.io/k8s-testimages/gcb-docker-gcloud:v20220830-45cbff55bc
     entrypoint: make
-    args:
-      - image-push
+#    args:
```

Comment:
Are you trying to fix the presubmit?

They are defined here: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

But if we change it, we somehow only need to do it for the main branch, as the older branchs don't have the `image-local-build` directive :/

Reply:
Yes, I'm trying to fix that.

> But if we change it, we somehow only need to do it for the main branch, as the older branchs don't have the image-local-build directive :/

It seems a bit hard...
I will figure out a way to not change the presubmit config.

Reply:
Maybe we can change the presubmit image to use the same image as the image build process?

The whole point of the presubmit is to make sure that the image will build :)

Reply:
That makes sense.
I will create a PR in test-infra repo.

Reply:
I have created a PR.
https://github.com/kubernetes/test-infra/pull/27411